src/main/webapp/resources/js/components/samples/components/SampleFileList.tsx

```diff
@@ -17,20 +17,31 @@ export function SampleFileList() {
   const dispatch = useDispatch();
   const [removeSampleFilesFromSample] = useRemoveSampleFilesMutation();
 
-  const { sample } = useSelector((state) => state.sampleReducer);
-  const { files } = useSelector((state) => state.sampleFilesReducer);
+  const { sample } = useSelector(
+    (state: RootStateOrAny) => state.sampleReducer
+  );
+  const { files } = useSelector(
+    (state: RootStateOrAny) => state.sampleFilesReducer
+  );
 
   /*
   Remove sequencing objects and/or genome assembly objects from sample
    */
-  const removeSampleFiles = ({ fileObjectId, type }) => {
+  const removeSampleFiles = ({
+    fileObjectId,
+    type,
+  }: {
+    fileObjectId: string | number;
+    type: string;
+  }) => {
     removeSampleFilesFromSample({
       sampleId: sample.identifier,
       fileObjectId,
       type,
     })
-      .then(({ data }) => {
-        notification.success({ message: data.message });
+      .unwrap()
+      .then(({ message }) => {
```

Comment:
Need type information.

Reply:
Updated in eb010bbe25ec4539fd70152d7567165863866208

Reply:
Ok.  But aren't all identifiers numbers on the server?  You could always cast them to a number if need be.

Reply:
Numbers on the server but come as strings in the response to the UI

Reply:
Updated in 445c58cec70fbfbbe6113426d30eb0caf860e90f